docs/troubleshooting.md

```diff
@@ -6,7 +6,6 @@ This page enumerates some common user facing issues around GCSFuse and also disc
 | Mount successful but files not visible                                                                                                                                                               | Try mounting the gcsfuse with --implicit-dir flag. Read the [semantics](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/docs/semantics.md) to know the reasoning.                                                                                                                                                                                                                                                                                                                                                                                                                                           |
 | Mount failed with fusermount3 exit status 1                                                                                                                                                          | It comes when the bucket is already mounted in a folder and we try to mount it again. You need to unmount first and then remount.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
-| Mount failed with error: Current requires cgo or $USER set in environment                                                                                                                            | It comes when we try mounting by building the gcsfuse codebase. To fix this, build the gcsfuse package by enabling the CGO_ENABLED flag in the go env and then mount back.<ol type="a"><li> Check the current value using - ```go env``` command. </li><li>If it is unset, set this using - ```export CGO_ENABLED=1``` command. </li></ol>                                                                                                                                                                                                                                                                             |
```

Comment:
This will probably have to be changed to an exact reverse troubleshooting step, where someone of forcibly building/running gcsfuse with `CGO_ENABLED=1` and they shouldn't. I just don't know what the exact wording should be.

Reply:
Changed.